macosx/TorrentCell.mm

```diff
@@ -101,9 +99,19 @@ - (instancetype)init
 
 - (id)copyWithZone:(NSZone*)zone
 {
-    id value = [super copyWithZone:zone];
-    [value setRepresentedObject:self.representedObject];
-    return value;
+    TorrentCell* copy = [super copyWithZone:zone];
+    copy->_fTitleAttributes = [_fTitleAttributes mutableCopyWithZone:zone];
+    copy->_fStatusAttributes = [_fStatusAttributes mutableCopyWithZone:zone];
```

Comment:
This is the core of the fix attempt: adding `mutablecopywithzone:` to those two mutable properties.